src/core/common/message.hpp

```diff
@@ -34,14 +34,17 @@
 #ifndef MESSAGE_HPP_
 #define MESSAGE_HPP_
 
-#include <stdint.h>
 #include <string.h>
 
 #include <openthread-types.h>
 #include <openthread-core-config.h>
 #include <common/code_utils.hpp>
 #include <mac/mac_frame.hpp>
 
+#ifdef _WIN32
+#pragma warning(disable:4201)  //nonstandard extension used : nameless struct/union
```

Comment:
It supports it, but again, in kernel mode, it throws a warning. I don't know why.